.github/workflows/dumb-action.yml

```diff
@@ -0,0 +1,35 @@
+name: Dumb action
```

Comment:
I like yours better( dumb action heh) but I didn't see it before creating the same thing.